/**:
  ros__parameters:
    stand_joint_angles: [0.12, 0.72, -1.45, -0.12, 0.72, -1.45, 0.12, 0.72, -1.45, -0.12, 0.72, -1.45]
    squat_joint_angles: [0.15, 1.30, -2.25, -0.15, 1.30, -2.25, 0.15, 1.30, -2.25, -0.15, 1.30, -2.25]
    command_rate: 50.0
    seconds_per_motion: 2.0
